homeassistant/components/dsmr/sensor.py

```diff
@@ -50,11 +53,7 @@
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the DSMR sensor."""
     # Suppress logging
-    logging.getLogger("dsmr_parser").setLevel(logging.ERROR)
-
-    from dsmr_parser import obis_references as obis_ref
-    from dsmr_parser.clients.protocol import create_dsmr_reader, create_tcp_dsmr_reader
-    import serial
+    _LOGGER.setLevel(logging.ERROR)
```

Comment:
This change doesn't look correct. We shouldn't set the level of our logger. It's the library logger that we want to change.

Reply:
I also found it wrong, but it was already so, just corrected the part of using the `_LOGGER` variable, instead of the direct call as it was. Do you think it is better to delete this line?

Reply:
No, I'm saying you have changed what logger you're targeting with this call. We need to restore the existing line.

Reply:
oh ok I got it

Reply:
Done